charts/api/templates/deployment-queue.yaml

```diff
@@ -144,7 +144,11 @@ spec:
           - name: GOOGLE_CLOUD_PROJECT_ID
             value: {{ .Values.app.gce.projectId | quote }}
           - name: GOOGLE_CLOUD_STORAGE_BUCKET
-            value: {{ .Values.app.gcsPublicStatic.gcsBucket | quote }}
+            valueFrom:
+              configMapKeyRef:
+                name: storage-bucket
+                key: gcs_api_static_bucket_name
+                optional: true
```

Comment:
This is set to optional so that a deployment of the API that doesn't use GCS (e.g. the one in our local dev env) doesn't fail because the configmap doesn't exist.